chapters/packages.tex

```diff
@@ -273,39 +267,39 @@ \subsection{External resources}\doublelabel{external-resources}
-package A in a directory A containing "package.mo") the resource
-"modelica://A/Resources/C.jpg" should be stored in the file
-"A/Resources/C.jpg", it is not recommend to use "modelica://A.B/C.jpg"
-for referencing resources; it could be stored in the file "A/B/C.jpg" -
-which is counter-intuitive if A.B is stored together with A. When
+package \lstinline!A! in a directory \filename{A} containing \filename{package.mo}) the resource
+\filename{modelica://A/Resources/C.jpg} should be stored in the file
+\filename{A/Resources/C.jpg}, it is not recommend to use \filename{modelica://A.B/C.jpg}
+for referencing resources; it could be stored in the file \filename{A/B/C.jpg} --
+which is counter-intuitive if \lstinline!A.B! is stored together with \lstinline!A!. When
 Modelica packages are stored in other formats a similar mapping should
 be defined, such that a resource with a given path should be unique for
 any package name that precedes it. The first part of the path may not be
 the name of a class in the package given by the authority. As above for
-"Modelica 3.2.1/package.mo" i.e. resources starting from "Modelica
-3.2.1", and "modelica://Modelica.Mechanics/C.jpg" is "Modelica
-3.2.1/Mechanics/C.jpg" - regardless of whether Modelica.Mechanics is
-stored in "Modelica 3.2.1/package.mo", "Modelica
-3.2.1/Mechanics/package.mo", or "Modelica 3.2.1/Mechanics.mo".
-
-For a Modelica-package stored as a single file, "A.mo", the resource
-"modelica://A/C.jpg" refers to a file "C.jpg" stored in the same
-directory as "A.mo", but using resources in this variant is not
+\filename{Modelica 3.2.1/package.mo} i.e. resources starting from \filename{Modelica
+3.2.1}, and \filename{modelica://Modelica.Mechanics/C.jpg} is \filename{Modelica
+3.2.1/Mechanics/C.jpg} - regardless of whether \lstinline!Modelica.Mechanics! is
+stored in \filename{Modelica 3.2.1/package.mo}, \filename{Modelica
+3.2.1/Mechanics/package.mo}, or \filename{Modelica 3.2.1/Mechanics.mo}.
+
+For a Modelica-package stored as a single file, \filename{A.mo}, the resource
+\filename{modelica://A/C.jpg} refers to a file \filename{C.jpg} stored in the same
+directory as \filename{A.mo}, but using resources in this variant is not
 recommended since multiple packages will share resources.
 
-In case the class-name contains quoted identifiers, the single-quote "`"
-and any reserved characters (``:'', ``/'', ``?'', ``\#'', ``{[}``,
-``{]}'', ``@'', ``!'', ``\$'', ``\&'', ``(``, ``)'', ``*'', ``+'',
-``,'', ``;'', ``='') should be percent-encoded as normal in URIs.
-
-{[}\emph{Example:}
+In case the class-name contains quoted identifiers, the single-quote `\lstinline!`!'
+and any reserved characters (`\lstinline!:!', `\lstinline!/!', `\lstinline!?!', `\lstinline!\#!', `\lstinline![!',
+`\lstinline!]!', `\lstinline!@!', `\lstinline!!!', `\lstinline!\$!', `\lstinline!\&!', `\lstinline!(`\lstinline!, `\lstinline!)!', `\lstinline!*!', `\lstinline!+!',
```

Comment:
`\lstinline!(`\lstinline!, must be an error. (the quotes were wrong before as well).

Reply:
Right.  I already fixed one of these, but missed this one.  Fixed.